documentation/docs/reference/pmm_components_and_versions.md

```diff
@@ -2,20 +2,20 @@
 
 The following table lists all the PMM client/server components and their versions:
 
-| PMM client/server component |Version | Documentation |Location on GitHub|
-|-----------------------------|--------|---------------|------------------|
-| Grafana  | 11.1.8*    | [Grafana Documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|
-| VictoriaMetrics| 1.93.4    | [VictoriaMetrics Documentation](https://docs.victoriametrics.com/)|[Github VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics)    |
-| Nginx    | 1.20.1 | [Nginx Documentation](http://nginx.org/en/docs/)|[Github Nginx](https://github.com/nginx/nginx)                                                    |
-| Percona Distribution for PostgreSQL  | 14.5    | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |
+| PMM client/server component | Version  | Documentation |Location on GitHub|
+|-----------------------------|----------|---------------|------------------|
+| Grafana  | 11.6.0*  | [Grafana Documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|
+| VictoriaMetrics| 1.114.0  | [VictoriaMetrics Documentation](https://docs.victoriametrics.com/)|[Github VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics)    |
+| Nginx    | 1.20.1   | [Nginx Documentation](http://nginx.org/en/docs/)|[Github Nginx](https://github.com/nginx/nginx)                                                    |
+| Percona Distribution for PostgreSQL  | 14.5     | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |
```

Comment:
https://www.percona.com/services/policies/percona-software-support-lifecycle shows  13, 14, 15, 16, 17 as supported DBs for PMM3

Reply:
Yeah, but inside we have PostgreSQL 14.5